lib/Echidna/Config.hs

```diff
@@ -95,6 +96,7 @@ instance FromJSON EConfigWithUsage where
         <*> v ..:? "corpusDir" ..!= Nothing
         <*> v ..:? "mutConsts" ..!= defaultMutationConsts
         <*> v ..:? "coverageReport" ..!= True
+        <*> v ..:? "coverageFormats" ..!= [Txt,Html,Lcov]
```

Comment:
I'm thinking if we should maybe merge this with `coverageReport` option. Like what should happen if `coverageReport` is `true` but `coverageFormats` is empty?